src/NotificationStack.vala

```diff
@@ -84,7 +83,12 @@ public class Gala.NotificationStack : Object {
          */
         update_positions (animate, notification.height);
 
-        move_window (notification, stack_x, stack_y + TOP_OFFSET + ADDITIONAL_MARGIN * scale);
+        var primary = display.get_primary_monitor ();
+        var area = display.get_workspace_manager ().get_active_workspace ().get_work_area_for_monitor (primary);
+
+        int notification_x_pos = area.width - window.get_frame_rect ().width;
```

Comment:
In my setup I have 2 monitors: |2|1|
At the moment the notification is being showed in my secondary monitor, this should fix it:

```suggestion
        int notification_x_pos = area.x + area.width - window.get_frame_rect ().width;
```